php/libraries/VisitTest.php

```diff
@@ -0,0 +1,380 @@
+<?php
+/**
+ * This file contains the unit test for Visit_Type class.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+require_once __DIR__ . "/NDB_Factory.class.inc";
+require_once __DIR__ . "/../../test/bootstrap.php";
+require_once __DIR__ . "/Visit.class.inc";
+require_once __DIR__ . "/Database.class.inc";
+require_once __DIR__ . "/testConfig.inc";
```

Comment:
Classes should generally be autoloaded, not required.

Reply:
was causing to much UnitTest problem and as it was supposed to be deleted before merge...